lab_1/var_15.py

```diff
@@ -0,0 +1,60 @@
+print('Lab_1')
+print('Var_15')
+print('-----------------------------------')
+
+
+# Generating array
+def generating_random_array(row, col):
+    # Random generating
+    from random import randint
+    arr = [[randint(-10, 10) for _ in range(col)] for _ in range(row)]
+    return arr
+
+
+# Printing array
+def print_array(arr):
+    print('Random array:')
+    print('-----------------------------------')
+    for i in arr:
+        for j in i:
+            print(f'{j:4}', end=' ')
+        print()
```

Comment:
don't use empty prints.
better to compose a str and print it just once after the loops